src/components/AttendantForm.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState, useEffect } from "react";
+import { getJobTitles, addAttendant, getAttendants } from "../api";
+import AttendantDisplay from "./AttendantDisplay";
+
+export default function AttendantForm() {
+  const [attendants, setAttendants] = useState([]);
+  const [name, setName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [jobTitle, setJobTitle] = useState("");
+  const [age, setAge] = useState("");
+  const [allJobTitles, setAllJobTitles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  async function submitHandler(e) {
+    e.preventDefault();
+    const submittedData = {
+      name,
+      lastName,
+      jobTitle,
+      age,
+    };
+
+    try {
+      setIsLoading(true);
+      const { status } = await addAttendant(submittedData);
+      if (status === 200) {
+        const { status: attendancesStatus, data } = await getAttendants();
+        if (attendancesStatus === 200) {
+          setAttendants([...data].sort((a, b) => a.age - b.age));
```

Comment:
The array spreading can be avoided since the data is already an array.
```suggestion
          setAttendants(data.sort((a, b) => a.age - b.age));
```